src/components/Message.svelte

```diff
@@ -15,6 +15,8 @@
   export let showTimestamp = false;
   export let thin = false;
   export let inanimate = true;
+  export let deleted = false;
+  export let messageArray = [];
```

Comment:
ig since this is convenient for the reactivity, we should keep it. However, it would be nice if it played well with the existing api. Perhaps maybe just have the default be null and then if it is null, set it to `message.messageArray` as a fallback.

Reply:
Done in latest commit